### PR TITLE
NISP-1781: Added new user friendly amount dissonance exclusion

### PR DESCRIPTION
**Pull request by dspork**
* Changed current exclusion page to old
* Added new page and added amount dissonance people to it
* Wrote dynamic content
* Using same exclusion message on both pages
* With @amyhart6

### CODE REVIEW

File: app/uk/gov/hmrc/nisp/views/excluded_sp_old.scala.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
+@*
```
Comment (roundrobin99) [inline app/uk/gov/hmrc/nisp/views/excluded_sp_old.scala.html]: should we consider changing the name of file?

File: app/uk/gov/hmrc/nisp/models/SPResponseModel.scala [hunk 2]
```diff
@@ -18,7 +18,7 @@ package uk.gov.hmrc.nisp.models
 
 import play.api.libs.json.Json
 
-case class SPResponseModel(spSummary: Option[SPSummaryModel], spExclusions: Option[ExclusionsModel] = None)
+case class SPResponseModel(spSummary: Option[SPSummaryModel], spExclusions: Option[ExclusionsModel] = None, niExclusions: Option[ExclusionsModel] = None)
```
Comment (roundrobin99) [inline app/uk/gov/hmrc/nisp/models/SPResponseModel.scala]: All arguments are coming as option, but we are not treating them as same way like -> Some(*)  or directly as niExclusions?

File: app/uk/gov/hmrc/nisp/controllers/ExclusionController.scala [hunk 3]
```diff
@@ -40,7 +41,13 @@ trait ExclusionController extends NispFrontendController with AuthorisedForNisp
   def showSP: Action[AnyContent] = AuthorisedByAny.async { implicit user => implicit request =>
     val nino = user.nino.getOrElse("")
     nispConnector.connectToGetSPResponse(nino).map {
-      case SPResponseModel(_, Some(spExclusions: ExclusionsModel)) => Ok(excluded_sp(nino, spExclusions))
+      case SPResponseModel(Some(spSummary: SPSummaryModel), Some(spExclusions: ExclusionsModel), niExclusionOption) => {
```
Comment (roundrobin99) [inline app/uk/gov/hmrc/nisp/controllers/ExclusionController.scala]: All arguments are coming as option, but we are not treating them as same way like -> Some(*) or directly as niExclusions?

**Comment by roundrobin99** [comment]
@InduTest - you can use this branch for testing.

**Comment by InduTest** [comment]
@roundrobin99 Ok

File: app/uk/gov/hmrc/nisp/controllers/ExclusionController.scala (hunk 3 above)
Comment (dspork) [inline app/uk/gov/hmrc/nisp/controllers/ExclusionController.scala]: @roundrobin99 The `case` is not dependent on the niExclusionOption having Some() or None, I need both. I've given it a name so I can check it in the code.

**Pull request merged**